packages/opentelemetry-exporter/src/index.js

```diff
@@ -0,0 +1,288 @@
+/*
+ * (c) Copyright IBM Corp. 2021
+ * (c) Copyright Instana Inc. and contributors 2021
+ */
+
+'use strict';
+
+const { backendConnector: instanaBackendConnector, environment: instanaEnvironment } = require('@instana/serverless');
+const { ExportResultCode } = require('@opentelemetry/core');
```

Comment:
The dependency from the `opentelemetry/core` is very small, as I am only using the enum of result codes.
I wonder if we could hard code these values to remove this dependency, having the risk that the default enum values can change.
On the other hand, a customer would probably be adding `opentelemetry/core` anyways. 
Having that said, we could also maybe use peerDependencies.

Any thoughts?

Reply:
Oof, good question. If we specify this as a dependency in package.json, we also prescribe a version number. Currently we pin it to 0.25.0. That is, if a customer uses 0.24.0 or 0.26.0, it would be installed two times anyway. Even if we go with a range, that still might happen.

Thus, the risk that the enum values that we use and the ones that the customer's app is using diverge exists no matter what we do, I don't see a way around that. But it should hopefully be a rather unlikely risk. 

I really don't have a strong opinion, but I would tend to leave the dependency in and not use a peer dependency.